src/components/cart/cartList.js

```diff
@@ -0,0 +1,140 @@
+import { useCart } from "./cartContext";
+import { useWishList } from "../WishList/wishContext";
+import { Header } from "../header";
+import "./cartStyles.css";
+import { ToastContainer } from "react-toastify";
+import {
+  wishlistHandler,
+  CartUpdate,
+  deleteCartItem
+} from "../ServerCalls/ServerCalls";
+import { isAddedInList } from "../utils/utils";
+import { EmptyCart } from "./EmptyCart";
+
+export const Cart = () => {
+  const { itemsInCart, dispatch: cartDispatch } = useCart();
+  const { dispatch: wishDispatch, wishList } = useWishList();
+  // const [isQtyUpdated, setQtyUpdate] = useState(true);
+  const totalPrice = () =>
+    itemsInCart.reduce((acc, item) => acc + item.price * item.quantity, 0);
+
+  return (
+    <section className="cart-container">
+      <div>
+        <Header />
+
+        {console.log("items in cart", { itemsInCart })}
+        <br />
+        <div className="aside cart-total center">
+          {itemsInCart.length !== 0 && (
+            <div className="bill">
+              <div className="para total">
+                <b>Total:</b> {""}
+                {""}
+                {totalPrice()}
+              </div>
+              <p className="para">
+                <strong>Delivery Charges</strong> :Free
+              </p>
+              <div className="para">
+                <b>Amount to be paid:</b>
+                {""}
+                {""}
+                {totalPrice()}
+              </div>
+              <button className="btn btn--primary btn--buy-now">
+                Order Now
+              </button>
+            </div>
+          )}
+        </div>
+        {itemsInCart.length === 0 ? (
+          <p className="para--lead">
+            <EmptyCart />
+          </p>
+        ) : (
+          <div className="container__main">
+            <h1 class="heading center fs-h1">My Cart</h1>
+            <div className="card-container">
+              {itemsInCart.map((data) => (
+                <div className="card card--display" Key={data._id}>
+                  <div className="card__thumbnail">
+                    <img
+                      src={data.imageUrl}
+                      className="card__img"
+                      alt="cardImg"
+                    />
+                  </div>
+                  <i
+                    className={
+                      isAddedInList(data._id, wishList)
+                        ? "fa fa-heart wish-icon wish-icon--selected"
+                        : "fa fa-heart wish-icon"
+                    }
+                    aria-hidden="true"
+                    onClick={() =>
+                      wishlistHandler(data, wishList, wishDispatch)
+                    }
+                  ></i>
+                  <div className="card__desc">
+                    <h1>
+                      <strong>{data.name}</strong>
+                    </h1>

```

Comment:
See you are using the same card component used in the product list page here. Extract it and use it as a separate component and reuse it wherever you want.